cirq-superstaq/cirq_superstaq/validation_test.py

```diff
@@ -27,3 +30,23 @@ def test_validate_cirq_circuits() -> None:
 
     with pytest.raises(ValueError, match="Circuit has no measurements to sample"):
         css.validation.validate_cirq_circuits(circuit, require_measurements=True)
```

Comment:
Could we add another check somewhere to make sure that the `validate_qubit_type()` function is being called - either by mocking the function or by validating a circuit with a bad qubit type.

Reply:
I changed the test below to call `validate_qubit_type() explicitly`, but regardless, the error would not have been raised in: https://github.com/Infleqtion/client-superstaq/blob/39506fe68c303abcf914a0aeef6f6a5069edbe66/cirq-superstaq/cirq_superstaq/validation_test.py#L51-L56
originally if `validate_qubit_type()` was not being called on `mock_cirq_circuit`.

Were you imagining something like https://github.com/Infleqtion/client-superstaq/pull/1050/commits/b1cf6220dada2a69f13b5259c1cdecd45ced09a0 for `test_validate_cirq_circuits()`?